code/index.html

```diff
@@ -2,11 +2,35 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project Name</title>
-    <!-- dont forget the link to the css file -->
-    <!-- dont forget the meta tag for responsiveness -->
+    <title>Kajsas form</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="https://kit.fontawesome.com/18157e8d5f.js" crossorigin="anonymous"></script>
   </head>
+  
   <body>
-    <!-- Build your site up as you want it! Make sure to use the form element and try to send that form to httpbin by adding action="https://httpbin.org/anything" and method="POST" to your form tag. -->
+    <div class="theFormBorder border-xs-none" >
+      <div class="theForm">
+        <h2>Sign up</h2>
+        <form action="http://httpbin.org/anything" method="POST">
+        <span class="fas fa-envelope icon"></span>
+        <input type="email" id="email" name="email" value="kajsa@kajsa.se" required><br>
```

Comment:
When you write `value="kajsa@kajsa.se"` - it makes it so that when I want to write something, the user first has to delete "kajsa@kajsa.se". If you want it as a placeholder, you could add this field instead:
`placeholder="kajsa@kajsa.se"`. Same goes for the password - you could have a placeholder called "Password" maybe?

Reply:
Try to remember to close all HTML-tags. They could either be self closing, like this `<input/>` or with opening and closing tags, like this `<input></input>`.

Reply:
This is totally optional, but when you have long lines - it's possible to split them up for better readability, like this:
```
<input
  type="email"
  id="email"
  name="email"
  value="kajsa@kajsa.se"
  required
/>
```

Reply:
> When you write `value="kajsa@kajsa.se"` - it makes it so that when I want to write something, the user first has to delete "[kajsa@kajsa.se](mailto:kajsa@kajsa.se)". If you want it as a placeholder, you could add this field instead:
> `placeholder="kajsa@kajsa.se"`. Same goes for the password - you could have a placeholder called "Password" maybe?

Ok. Thanks for your input. =)

Reply:
> Hi Kajsa!
> Wow - I'm super impressed by the result of your form - it looks really neat and professional! I wrote some comments that you could take a look at. If you want to - you can just continue with this project if you want to try out checkboxes or something as well - but as I said - this looks super nice! Really nice that you pulled of the icons!
> 
> One thing to think about is the classnames / id. They should be descriptive - like you have "password" and "button". It's really nice if they are all written in the same case style. When you write like this: `theForm` - it's called camelCase, and when you write like `border-xs-none` - it's called kebab-case. [Here](https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841)'s an article where you can read a bit about it. Standard for CSS classes is to use kebab-case - but pick one and stick to it!
> 
> Have a good weekend and a great summer! ☀️

Thanks for your comments. I am grateful for your help and tips.
Wish you a nice continued summer.
/Kajsa